src/main/java/net/fabricmc/tinyremapper/api/WrapperFunction.java

```diff
@@ -0,0 +1,7 @@
+package net.fabricmc.tinyremapper.api;
+
+import org.objectweb.asm.ClassVisitor;
+
+public interface WrapperFunction {
```

Comment:
Maybe add documentation on how this one should be used.

Reply:
Maybe also consider adding `@FunctionalInterface` for this interface.

Reply:
Should add a simple API like `andThen` to chain wrappers easily, and use that to support multiple wrappers in remapper builder code